src/services/formatting/formatting.ts

```diff
@@ -438,6 +438,25 @@ namespace ts.formatting {
             }
         }
 
+        if (previousRange! && formattingScanner.getStartPos() >= originalRange.end) {
+            const token =
+                formattingScanner.isOnEOF() ? formattingScanner.readEOFTokenRange() :
+                formattingScanner.isOnToken() ? formattingScanner.readTokenInfo(enclosingNode).token :
+                undefined;
+
+            if (token) {
+                processPair(
+                    token,
+                    sourceFile.getLineAndCharacterOfPosition(token.pos).line,
+                    enclosingNode,
+                    previousRange,
+                    previousRangeStartLine!,
+                    previousParent!,
+                    enclosingNode,
+                    /*dynamicIndentation*/ undefined);

```

Comment:
I honestly have no idea what the right value for this is and if it ever matters, but it didn’t matter for any existing tests, so I made it optional and skipped line/indentation edits in the implementation if it’s not passed.